### PR TITLE
mbeddr.doc: don't stop screenshot generation in MpsEnvironment

### PR DESCRIPTION
**Pull request by arimer**
Bringing back screenshooter from CL builds. Kept the warning to switch to build-backends.

### CODE REVIEW

**Comment by alexanderpann** [comment]
The change looks fine to me but I don't understand the implications. Why was the break after the if statement anyway?

**Comment by sergej-koscejev** [comment]
> Why was the break after the if statement anyway?

In our customer project screenshots could not be correctly generated in MpsEnvironment so there was no point in iterating through all of them and printing the message for each one. @arimer believes that in his case MpsEnvironment could work.

**Review by mgronover — DISMISSED** [review]
LGTM

File: code/languages/com.mbeddr.doc/languages/com.mbeddr.doc/languageModels/plugin.mps [hunk 1]
```diff
@@ -2318,7 +2317,7 @@
                 <node concept="17QB3L" id="5Ulr5FUQKky" role="1tU5fm" />
                 <node concept="3cpWs3" id="5Ulr5FUQLHz" role="33vP2m">
                   <node concept="Xl_RD" id="5Ulr5FUQLH$" role="3uHU7B">
-                    <property role="Xl_RC" value="Screenshots cannot be taken in MpsEnvironment (Ant &lt;generate&gt; task). " />
+                    <property role="Xl_RC" value="Screenshots might not be taken in MpsEnvironment (Ant &lt;generate&gt; task). " />
```
Comment (sergej-koscejev) [inline code/languages/com.mbeddr.doc/languages/com.mbeddr.doc/languageModels/plugin.mps:2320]:
```suggestion
                    <property role="Xl_RC" value="Screenshots might not be taken correctly in MpsEnvironment (Ant &lt;generate&gt; task). " />
```

**Comment by arimer** [comment]
We will check verify after the merge on customer side if CL screen-shooting works. In case it is not, I will revert my commits.

**Comment by sergej-koscejev** [comment]
Can you verify it with a local build?

**Comment by arimer** [comment]
Now that we can [publish snapshots](https://github.com/mbeddr/mbeddr.core/pull/2515), we will test that code on a snapshot artifact first.

**Review by alexanderpann — APPROVED** [review]
Let's merge it, since we can't test it locally. We can still revert it, if it doesn't work out.

**Comment by arimer** [comment]
Discussed with @alexanderpann. We merge and revert in case it is not working.

@sergej-koscejev 
> Can you verify it with a local build?

Unfortunatelly not :(

**Comment by arimer** [comment]
So I tried the CL screen-shooter on a command line build on mac and linux. It works on mac (with matching jbr) but fails on linux (with matching jbr). In case you want to try it out, have a look at [iest3.core](https://github.com/IETS3/iets3.core/tree/debug/mbeddrDocScreenShoot)

**Pull request merged**